### ISSUE TITLE
Change "input" to "decode" and add an input to it? (Node-RED)

### ISSUE THREAD

**Issue by spants** [enocean-js/enocean-js#133]
Just wondering if you could separate the serial comms input and the decoding part on the "input" node. We can then use the "tcp in" node or "node-red-node-serialport" nodes as input for the new "decoder" node.

**Comment by Holger-Will** [comment]
in the input, no real decoding is happening. It just uses the output of the serialport parser.

### tl;dr

I can make the actor accept the raw string as well as a raw buffer. can you post the raw msg object you are getting from the tcp-in node here?

### workaround

for now, you can make it work by adding a bit of metadata to your message:

```
  "meta": {
    "type": "radio-erp1"
  }
```

### details

consider this telegram: `55000a0701eba50000225d01858d920001ffffffff430022`

the first 5 bytes or 10 characters are the headers of the telegram telling us things about the content. so  `55.00.0a.07.01` is the header. the last byte `01` tells us about the telegram type, in this case, it is a radio-erp1 packet. the actor only understands radio-erp1 telegrams but instead of looking at the passed string, I pass some metadata to the actor node and look at that... this is not really needed though, because the information is already there. So it is quite straight forward to change this...

**Comment by spants** [comment]
That works - thanks! 
Might be useful to remove that requirement if not needed.

**Comment by Holger-Will** [comment]
with `v0.7.2` you can now send raw telegram strings without meta information
